src/test/unit/oceanP2P.test.ts

```diff
@@ -55,6 +56,7 @@ describe('OceanP2P Test', () => {
     config2.p2pConfig.bootstrapNodes = []
     // enable private IP
     config2.p2pConfig.announcePrivateIp = true
+    config1.p2pConfig.filterAnnouncedAddresses = ['172.15.0.0/24'] // allow nodes to see each other locally for tests
```

Comment:
shouldn't be config2 ?

```suggestion
    config2.p2pConfig.filterAnnouncedAddresses = ['172.15.0.0/24'] // allow nodes to see each other locally for tests
```

Reply:
yeah :-) .... damn copy/paste